source/components/RadialListItem/index.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import cxs from 'cxs'
+
+const BASE_STYLES = {
+  display: 'block',
+  zIndex: 2,
+  position: 'absolute',
+  top: '50%',
+  left: '50%',
+  width: '80%',
+  height: 20,
+  margin: 0,
+  padding: 0,
+  listStyle: 'none',
+  transformOrigin: '0% 50%'
+}
+
+const SPAN_STYLES = {
+  display: 'block',
+  position: 'absolute',
+  top: 0,
+  width: '100%',
+  textAlign: 'center'
+}
+
+const RadialListItem = ({
+  item,
+  index,
+  count,
+  offsetDegrees,
+  itemStyles
+}) => {
+  const rotation = offsetDegrees + (360 / count * index)
+  const listItemStyles = Object.assign({}, BASE_STYLES, itemStyles, {
+    transform: `rotate(${rotation}deg)`
+  })
+  const spanStyles = Object.assign({}, SPAN_STYLES, {
+    transform: `rotate(${-rotation}deg)`
+  })
```

Comment:
Just a little exercise in functional programming:

```
const rotate = (rotation) => Object.assign.bind({}, { transform: `rotate${rotation}` })
const position = offsetDegrees + (360 / count * index)
const rotation = rotate(position) // this could curried to be rotation = (position) => rotate(position) but no ramda
const negativeRotation = rotate(-position)

const spanStyles = rotation(SPAN_STYLES)
const listItemStyles = negativeRotation(BASE_STYLES, itemStyles)
```